UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/Bodypart/BodyPartComponentBase.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Linq;
+
+namespace HealthV2.Living.PolymorphicSystems.Bodypart
+{
+	public abstract class BodyPartComponentBase : BodyPartFunctionality
+	{
+		public override void RemovedFromBody(LivingHealthMasterBase livingHealth)
+		{
+			foreach (var sys in livingHealth.ActiveSystems)
+			{
+				sys.BodyPartRemoved(RelatedPart);
+			}
+		}
+
+		public override void AddedToBody(LivingHealthMasterBase livingHealth)
```

Comment:
`OnAddedToBody()`

Reply:
this isn't a new function though?, I can change it if you want